### PR TITLE
Fix #36576 gnatkovsky.com.ua

### LINKED ISSUE #36576: gnatkovsky.com.ua
### Issue URL (Anti Adblock Script)
[http://gnatkovsky.com.ua/poyavlenie-bloka-pri-navedenii-s-pomoshhyu-css.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fgnatkovsky.com.ua%2Fpoyavlenie-bloka-pri-navedenii-s-pomoshhyu-css.html)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.1.1
Browser: | YaBrowser
Filters: | AdGuard Russian,<br>AdGuard Turkish,<br>AdGuard Base,<br>AdGuard German,<br>AdGuard Dutch
Other extensions used: | не имеются

### PR DESCRIPTION
**Pull request by tvinzz**
https://github.com/AdguardTeam/AdguardFilters/issues/36576

### CODE REVIEW

File: RussianFilter/sections/specific.txt [hunk 1]
```diff
@@ -18,6 +18,7 @@ Ly9hbi55YW5kZXgucnUv
 ! adbetnet domains
 ||cdn.*/libs/b.js$third-party,domain=nnm-club.lib|nnm-club.me|nnm-club.tv|nnmclub.to|seasonvar.ru|mintmanga.com|realmadrid.ru
 !
+gnatkovsky.com.ua##.r_sell > a[target="_blank"]
```
Comment (BlazDT) [inline RussianFilter/sections/specific.txt]: .r_sell is enough in this case.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/specific.txt (hunk 1 above)
Comment (Alex-302) [reply RussianFilter/sections/specific.txt]: or add domain to `/images/banner-$domain=erospirit.ru|sudba.in.ua`

File: RussianFilter/sections/antiadblock.txt [hunk 2]
```diff
@@ -33,6 +33,8 @@ soskidkami.ru#%#AG_abortOnPropertyRead('anOptions');
 ! AdBlocksBuilder
 .com/resources/draw_script.js$third-party
 !
+https://github.com/AdguardTeam/AdguardFilters/issues/36576
+||gnatkovsky.com.ua/*/single.js
```
Comment (Alex-302) [inline RussianFilter/sections/antiadblock.txt]: it contains not only anti-adblock

**Review by Alex-302 — CHANGES_REQUESTED** [review]
and missed social share leftover `gnatkovsky.com.ua##.sh_this`

**Comment by tvinzz** [comment]
1. single.js checks the height of `.no-adb` and if it's less then `1`, antiadblock works.
2. removed sharing button

**Comment by Alex-302** [comment]
> 1. single.js checks the height of `.no-adb`

contains. And other functions of the site.

File: RussianFilter/sections/antiadblock.txt [hunk 3]
```diff
@@ -54,6 +52,9 @@ sostav.ru###rek_blocker
 ! https://forum.adguard.com/index.php?threads/32223/
 drive2.ru$$script[wildcard="*eval(decodeURIComponent*BQCkh6XABTS*"]
 drive2.ru#%#AG_abortInlineScript(/BQCkh6XABTS/, 'eval');
+! https://github.com/AdguardTeam/AdguardFilters/issues/36576
```
Comment (Alex-302) [inline RussianFilter/sections/antiadblock.txt]: а хинты? Для Сафари и CB есть `content_blocker.txt` в англ. фильтре

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Comment by tvinzz** [comment]
replaced $generchide rule with
```
gnatkovsky.com.ua#@#.adsbygoogle
gnatkovsky.com.ua#$#.adsbygoogle { height: 1px!important; }
```

**Review by slavaleleka — APPROVED** [review]

**Pull request merged**